### PR TITLE
Fixed shifting details page responsiveness and add shadcn Breadcrump

### LINKED ISSUE #8831: phone size responsive issue shifting details page
**Describe the bug**
responsive issue on shifting details page, 

**To Reproduce**

1. Go to 'shifting on sidebar'
2. Click on 'all details button from any card and head to shift details page'
                  'in phone size responsive issue'

**Expected behavior**
clear and better mobile user experience

**Screenshots**
Image

**Desktop (please complete the following information):**

- OS: [windows]
- Browser [chrome]
- Version [129.0.6668.101]

**Additional context**

### PR DESCRIPTION
**Pull request by modamaan**
## Proposed Changes
change the button alignment when phone size

- Fixes #8831 

Before
Screenshot 2024-10-20 000900

After
Screenshot 2024-10-20 011739

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by modamaan** [comment]
shadcn added https://ui.shadcn.com/docs/components/breadcrumb

https://github.com/user-attachments/assets/0ae0e78c-5673-4265-a7d5-69ce8e02eb87

**Review by bodhish — CHANGES_REQUESTED** [review]
- [ ] Fix all lint issues, also check if your auto format is working 
- [ ] Remove unused variables and functions, this should have been caught before committing in pre commit hook 🤔. Check your setup 
- [ ] Lets have one behaviour both desktop and mobile, keep it as close as how Shadcn works. 
- [ ] UI should match, from the video I can quickly spot that you are missing a `>` at the end
image

**Comment by modamaan** [comment]
Screenshot 2024-10-22 095006
Screenshot 2024-10-22 095502

there is no ` <BreadcrumbSeparator />` shadcn at the end,  If you'd like let me know

**Comment by bodhish** [comment]
on the video you shared I can see that we are missing a `>` after the `...`
image

**Comment by modamaan** [comment]
@bodhish @vigneshhari 

Screenshot 2024-10-23 001317

File: src/Components/Common/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,20 +1,11 @@
+/* eslint-disable prettier/prettier */
```
Comment (rithviknishad) [inline src/Components/Common/Breadcrumbs.tsx]: why disable prettier?

Comment (modamaan) [reply src/Components/Common/Breadcrumbs.tsx]: Changed @rithviknishad

**Comment by modamaan** [comment]
@bodhish how to resolve conflict issue

**Comment by rithviknishad** [comment]
https://github.com/ohcnetwork/care_fe/pull/8875#issuecomment-2431773504

**Comment by modamaan** [comment]
@rithviknishad can you check the updates

File: src/Components/Shifting/ShiftDetailsUpdate.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import * as Notification from "../../Utils/Notifications.js";
+import * as Notification from '../../Utils/Notifications.js';
```
Comment (bodhish) [inline src/Components/Shifting/ShiftDetailsUpdate.tsx]: We use `""` instead of `''`

**Review by bodhish — CHANGES_REQUESTED** [review]
You missed to implmenet 
- [ ] Fix all lint issues, also check if your auto format is working. (How did the lint miss catching the quotes issue 🤔)
- [ ] Lets have one behaviour both desktop and mobile, keep it as close as how Shadcn works.
Its having the old behavior in desktop and new one in mobile screens. 

### Other Changes 
- [ ] `...` is having a different color, it look alike right 🤔 

image

- [ ] Login seems to be broken for showing the last item. 
### Before
image

### after

image

- [ ] Somehow home button lost an underline; 
- [ ] why is logic with `MENU_TAGS` removed?

before requesting the review make sure all these changes are tested.

**Comment by modamaan** [comment]
Screenshot 2024-10-25 010002

updated  phone and desktop

File: src/Components/ui/breadcrumb.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
+import * as React from "react";
```
Comment (bodhish) [inline src/Components/ui/breadcrumb.tsx:1]: The folder casing for the file is wrong.

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/ui/breadcrumb.tsx (hunk 3 above)
Comment (modamaan) [reply src/Components/ui/breadcrumb.tsx:1]: Can you explain @bodhish

Comment (modamaan) [reply src/Components/ui/breadcrumb.tsx:1]: my  folder src/Components/ui/breadcrumb.tsx, but real folder changed to components (case changed), how can i update that changes in my vsCode, when i chainge code and try git status it shows src/Components/ui/breadcrumb.tsx  @bodhish

Comment (modamaan) [reply src/Components/ui/breadcrumb.tsx:1]: i cant change the folder path, i try to pull the code, but didnt work

**Pull request closed without merging**